thrift/binary.go

```diff
@@ -204,10 +204,14 @@ func (p BinaryProtocol) UnwrapBody() (string, TMessageType, int32, FieldID, []by
 		return name, rTyp, seqID, 0, nil, err
 	}
 	// read the success struct
-	_, _, structID, err := p.ReadFieldBegin()
+	_, typeId, structID, err := p.ReadFieldBegin()
 	if err != nil {
 		return name, rTyp, seqID, structID, nil, err
 	}
+	//empty response
+	if typeId == STOP {
```

Comment:
what's the reason to have nil body in READ WORLD?

Reply:
> what's the reason to have nil body in READ WORLD?

Unintentionally returned a nil resp

Reply:
> what's the reason to have nil body in READ WORLD?

btw it seems returning a nil body will not trigger error in other kitex thrift rpc call